queries.sql

```diff
@@ -138,4 +138,75 @@ WHERE escape_attempts = 0 AND owner_id = 5;
 SELECT full_name, COUNT(owner_id) AS Total_animals FROM owners
 JOIN animals ON  owners.id = animals.owner_id
 GROUP BY full_name
-ORDER BY COUNT(name) DESC;
\ No newline at end of file
+ORDER BY COUNT(name) DESC;
+
+-- Update Day 4 week 1 queries
+-- Update who was the last animal seen by William Tatcher
+SELECT (animals.name) AS animalName, (vets.name) AS vetsName, (visits.date_of_visit) AS dateVisited
+FROM animals
+JOIN visits ON animals.id = visits.animals_id
+JOIN vets ON vets.id = visits.vets_id
+WHERE vets_id = 1
+ORDER BY (visits.date_of_visit) DESC
+LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(visits.vets_id) AS Different_animals_seen, (vets.name) AS vets_name
+FROM visits
+JOIN vets ON visits.vets_id = vets.id
+WHERE vets.id = 3
+GROUP BY vets.name;
```

Comment:
- [ ] The instruction is to write a query to get the amount of DIFFERENT animals that Stephanie Mendez saw. At the moment, your query is not doing that. Please ensure to write the correct query

Reply:
Hello @AnselemOdims  This is what I get from running the query. The question asks "How many different animals did Stephanie Mendez see?" and the query returns 4 which is the number of different animals that Stephanie attended to. 
Is this wrong or was there an error in the review?
![vet](https://user-images.githubusercontent.com/86318284/171791021-fd12c23e-11fb-48b5-a751-cbd8e4e0a53f.png)